matrix.js

```diff
@@ -0,0 +1,26 @@
+export class Matrix {
+  constructor(matrixString) {
+    this.matrixString = matrixString;
+    this.rows = this.getMatrixInRowForm();
```

Comment:
`rows` doesn't belong in constructor. It is a behaviour for Matrix object. Read about designing your classes in terms of objects and their behaviours. How do you model classes etc.
Same goes for `columns`.

Reply:
Ask any doubt if you have any. Doubts are welcome always !!